exploit/utils.js

```diff
@@ -243,6 +243,9 @@ exports.trunclt32 = function (num, bits) {
 exports.trunclt64 = function (num, bits) {
 	num = this.pad64(num);
 	if(this.bits <= 32) {
+		if (num[1] !== 0) {
+			throw new Error("number is too large for " + bits + "bits");
+		}
 		return [this.trunclt32(num), 0];
```

Comment:
this invocation of `trunclt32` looks to be wrong. can you fix that too?

Reply:
Thanks for the review, mission2000.

Can you help me understand the problem with the invocation of trunclt32?
It appears that trunclt32 can accept an array.
It appears that trunclt32 allows bits parameter to be 32.
It appears that trunclt32 ensures the return value is a single number (not array).
It appears that trunclt64 ensures the return value is an array of two numbers.

Actually, looking at this more, it appears there is no bug, it's just that trunclt64 allows trunclt32 to handle the exception reporting when the high bits are set.  So, I'm happy to close this PR, and discard these changes.